modules/lib/maintainers.nix

```diff
@@ -305,4 +305,10 @@
     github = "lukasngl";
     githubId = 69244516;
   };
+  gvolpe = {
+    name = "Gabriel Volpe";
+    email = "volpegabriel@gmail.com";
+    github = "gvolpe";
+    githubId = 443978;
+  };
```

Comment:
Since you are already in Nixpkgs' `maintainers/maintainer-list.nix` file you don't need to be here:

```suggestion
```